test/tests/p5.Gain.js

```diff
@@ -0,0 +1,51 @@
+const expect = chai.expect;
+let gain;
+
+describe('p5.Gain', function () {
+  beforeEach(function () {
+    gain = new p5.Gain();
+  });
+
+  it('can be created', function () {
+    expect(gain.input).to.have.property('gain');
+    expect(gain.output).to.have.property('gain');
+    let audioContext = gain.ac;
+    expect(audioContext).to.have.property('baseLatency').to.be.an('number');
+    expect(audioContext).to.have.property('destination');
+    expect(audioContext).to.have.property('state').to.be.an('string');
```

Comment:
these look like the same `expect`'s that were part of the AudioContext test. I wonder if they could be abstracted to a method wherever we need to verify that there is an AudioContext.

Reply:
Yes, actually. We would use that very often. also we can add similar functions for checking gain nodes and etc.

Will add it if I got time or will at least add a issue/TODO regarding this.